orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/ExpressionAware.kt

```diff
@@ -49,7 +50,7 @@ interface ExpressionAware {
           }
         }
 
-        val result = processed[key] ?: execution.getContext()[key]
+        val result = processed[key]
```

Comment:
No need to do this as `processed` is now a `StageContext` so it will do it anyway.